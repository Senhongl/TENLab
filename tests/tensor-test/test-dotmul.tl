a = [[1, 2, 3], [4, 5, 6]];
b = [[-1, -2, -3], [-4, -5, -6]];
c = a .* b;
print(c);
d = a .* 2;
print(d);
e = float_of(a) .* 0.2;
print(e);