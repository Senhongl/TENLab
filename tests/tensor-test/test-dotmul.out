-1 -4 -9
-16 -25 -36
[ CPUIntType{2,3} ]
  2   4   6
  8  10  12
[ CPUIntType{2,3} ]
 0.2000  0.4000  0.6000
 0.8000  1.0000  1.2000
[ CPUDoubleType{2,3} ]
